static/src/javascripts/projects/common/modules/commercial/build-page-targeting.spec.ts

```diff
@@ -291,7 +291,10 @@ describe('Build Page Targeting', () => {
 	});
 
 	it('should remove empty values', () => {
-		window.guardian.config.page = {} as PageConfig;
+		window.guardian.config.page = {
+			// pageId should always be defined
+			pageId: 'football/series/footballweekly',
+		} as PageConfig;
```

Comment:
This test started failing since the new targeting expects `pageId` to always be a `/${string}`, so coercing this empty targeting object for the sake of this test gives `urlkw: [ 'undefined' ]`. Since `pageId` comes through on the CAPI object, it makes sense to be defined as part of the test as well.